assets/src/dashboard/app/api/test/useTemplateApi.js

```diff
@@ -17,7 +17,10 @@
 /**
  * Internal dependencies
  */
-import { toUTCDate } from '../../../../date';
+/**
+ * External dependencies
+ */
+import { toUTCDate } from '@web-stories-wp/date';
 import { reshapeTemplateObject } from '../useTemplateApi';
```

Comment:
Need to fix these comments here (and elsewhere):

```js
/**
 * External dependencies
 */
import { toUTCDate } from '@web-stories-wp/date';

/**
 * Internal dependencies
 */
import { reshapeTemplateObject } from '../useTemplateApi';
```

Reply:
Fixes https://github.com/google/web-stories-wp/pull/6193/commits/31c5ea5c5b5027b045ffa3f23e87a75ac95685e4